docs/backend/content-types/index.md

```diff
@@ -12,15 +12,223 @@ myst:
 # Content Types
 
 ```{seealso}
-See the chapter {ref}`training:dexterity1-label` from the Mastering Plone 6 Training.
+See the chapter {ref}`training:dexterity1-label` from the Mastering Plone 6 Training for a step-by-step tutorial to create a custom content type.
 ```
 
-```{todo}
-Contribute to this documentation!
-See issue [Backend > Content Types needs content](https://github.com/plone/documentation/issues/1303).
+## What is a content type?
+
+Each item in a Plone site is an instance of a particular content type. We have different content types to reflect the different kinds of information about which we need to collect and display information.
```

Comment:
One sentence per line, here and elsewhere. I'll just make the one comment for this style guide matter.

```suggestion
Each item in a Plone site is an instance of a particular content type.
We have different content types to reflect the different kinds of information about which we need to collect and display information.
```